app/livechat/imports/server/rest/sms.js

```diff
@@ -70,8 +70,9 @@ API.v1.addRoute('livechat/sms-incoming/:service', {
 	post() {
 		const SMSService = SMS.getService(this.urlParams.service);
 		const sms = SMSService.parse(this.bodyParams);
+		const { departmentID } = this.queryParams;
```

Comment:
@bhardwajaditya Is this query param departmentId (i.e. `pDSPNogBNMGv5iAtX `) or departmentName(i.e. `Viasat Customer Support`) ?

If it's `Viasat Customer Support` we should rename it to `departmentName` to avoid confusion

Reply:
Changed this.